addons/mass_mailing/views/mailing_list_views.xml

```diff
@@ -169,7 +169,7 @@
                             </div>
                         </div>
                         <!-- Card when the Kanban view is not grouped -->
-                        <div class="oe_kanban_global_click o_mailing_list_kanban_ungrouped d-flex flex-row justify-content-between align-items-center flex-wrap">
+                        <div class="oe_kanban_global_click o_mailing_list_kanban_ungrouped d-flex gap-1 flex-row justify-content-between align-items-center flex-wrap">
```

Comment:
I've the feeling than buttons are still too squashed with the rest, what do you think ? 
![Capture d’écran 2023-06-30 à 11 06 15](https://github.com/odoo/odoo/assets/80679690/532de474-17f2-4a89-b71f-98f9831c4cb7)

Reply:
Also, can you rebase your branch ?

Reply:
I agree.
I've done another fix for this, but I moved the buttons on tablet because the balance seems of to me.

<img width="789" alt="Screenshot 2023-06-30 at 16 56 55" src="https://github.com/odoo/odoo/assets/19491443/cfb6e1a0-0676-497f-9dbc-b4911ac0f477">

wdyt?